src/server/object_services/schemas/object_part_indexes.js

```diff
@@ -12,6 +12,7 @@ module.exports = [{
             start: 1,
         },
         options: {
+            name: 'undeleted_parts_by_objects_and_offset_index',
```

Comment:
Can you verify that providing a name to existing index will indeed set its name?

Reply:
BTW I don't think we need the name to say "undeleted" explicitly, I would keep it as "parts_by_objects_and_offset_index"

Reply:
It doesn't set his name - changed to the default old name 'obj_1_start_1'